db/migrate/20150709221755_create_logins.rb

```diff
@@ -1,14 +1,17 @@
 class CreateLogins < ActiveRecord::Migration
 
   def change
-    create_table :logins do |t|
+    primary_key_chooser = lambda do |key|
+      RailsApiAuth.primary_key_type ? { key => RailsApiAuth.primary_key_type } : {}
+    end
```

Comment:
That's true would make more sense

On Thu, 15 Oct 2015 13:49 Marco Otte-Witte <notifications@github.com> wrote:

> In db/migrate/20150709221755_create_logins.rb
> <https://github.com/simplabs/rails_api_auth/pull/23#discussion_r42119291>:
>
> > @@ -1,14 +1,17 @@
> >  class CreateLogins < ActiveRecord::Migration
> >
> >    def change
> > -    create_table :logins do |t|
> > +    primary_key_chooser = lambda do |key|
> > +      RailsApiAuth.primary_key_type ? { key => RailsApiAuth.primary_key_type } : {}
> > +    end
>
> would be cool to have this in a private method, e.g. def
> key_option(option_name)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simplabs/rails_api_auth/pull/23/files#r42119291>.
>